docker-images/operator/scripts/launch_java.sh

```diff
@@ -14,6 +14,11 @@ function get_gc_opts {
 
 export MALLOC_ARENA_MAX=2
 
+if [[ -z "$STRIMZI_OPERATOR_NAME" ]]; then
+  STRIMZI_OPERATOR_NAME="$(hostname)"
+  export STRIMZI_OPERATOR_NAME
+fi
```

Comment:
@scholzj How do you feel about this? If the env var STRIMZI_OPERATOR_NAME isn't set, then use `hostname` to populate it. 

It leaves the door open for end users to set the operator name to something that has more meaning to them than just a pod name, perhaps a name describing the cluster that the given Strimzi is managing e.g., "low-latency-cluster-operator" or "logging-data-cluster-operator". 

Too fancy?

Reply:
It is good in general. Not sure it is needed for fancy hostnames - but it is useful to let the user configure it optionally at least as a fallback in case of some issued. Two comments to this:
* Following the discussion about having a generic tool for setting the events and having it in operator-common, where we always concluded that we keep it for now cluster operator specific, this should be then in `cluster_operator_run.sh` and not in `launch_java.sh`
* I guess `hostname` is not installed there. So you either need to install it or maybe better use `/proc/sys/kernel/hostname` instead?